index.html

```diff
@@ -23,23 +23,32 @@
           <a href="#" id="my-logo">RolandM</a>
         </div>
         <div class="menu">
+          
           <nav class="nav-bar">
-            <div class="bar-menu">
+         <!-- <div class="bar-menu">
```

Comment:
- [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.